### PR TITLE
single quote enforcement

### LINKED ISSUE #75: single quote enforcement
we've agreed upon internally that we should use a single set of quotes throughout our files to avoid ending up with files with mixed usage.

### PR DESCRIPTION
**Pull request by rlgomes**
fixes #75

enforce single quote usage

### CODE REVIEW

**Comment by rlgomes** [comment]
@juttle/developers review please

File: .eslintrc [hunk 1]
```diff
@@ -8,7 +8,8 @@
         "indent": [ 2, 4, { "SwitchCase": 1 } ],
         "linebreak-style": [ 2, "unix" ],
         "no-unused-vars": [2, { "args": "none" }],
-        "semi": [ 0 ]
+        "semi": [ 0 ],
```
Comment (mnibecker) [inline .eslintrc]: this means we're requiring semi colons, right?

Comment (mnibecker) [inline .eslintrc]: wait nvm

**Comment by mnibecker** [comment]
+1

**Pull request merged**